#OpenXT:Networkmanager checks the uid of the processes for permissions.
#       This doesn't work for nm-applet as it is connecting over argo.
#       Hence assume uid to be 0(root).
--- a/src/nm-bus-manager.c
+++ b/src/nm-bus-manager.c
@@ -417,18 +417,7 @@ _bus_get_unix_user (NMBusManager *self,
 	guint32 unix_uid = G_MAXUINT32;
 	gs_unref_variant GVariant *ret = NULL;
 
-	ret = _nm_dbus_proxy_call_sync (NM_BUS_MANAGER_GET_PRIVATE (self)->proxy,
-	                                "GetConnectionUnixUser",
-	                                g_variant_new ("(s)", sender),
-	                                G_VARIANT_TYPE ("(u)"),
-	                                G_DBUS_CALL_FLAGS_NONE, 2000,
-	                                NULL, error);
-	if (!ret)
-		return FALSE;
-
-	g_variant_get (ret, "(u)", &unix_uid);
-
-	*out_user = (gulong) unix_uid;
+	*out_user = (gulong) 0;
 	return TRUE;
 }
 
